integr8ly/application-monitoring-operator
# Bump Prometheus operator to v0.38.1

## Description
[MGDAPI-808](https://issues.redhat.com/browse/MGDAPI-808)

Bumps the Prometheus Operator to `v0.38.1` and the Prometheus version to `2.15.2` which aligns with the `openshift-monitoring` Prometheus versions.

## Verification Steps:
-  Install using this branch:
```
 make cluster/install/local
 make code/run
```
- Once an install is complete, check both the Prometheus Operator and Prometheus pods to make sure there are no errors. Also, check the events.
- Ensure the Prometheus Operator version installed is `v0.38.1` (you can check the Prometheus Operator pod logs or deployment image spec for this)
- Navigate to the Prometheus route In the `application-monitoring` namespace -> status from the top menu -> Runtime and Build information. Ensure the version is `2.15.2`:

![build](https://user-images.githubusercontent.com/47980969/101922781-7193b100-3bc6-11eb-9537-bcea2ffa66d0.png)
- Do the same with the `openshift-monitoring` Prometheus to make sure the versions match up (both should be `2.15.2`)
- Ensure that you can also login to the alertmanager with any issue, as a minor change was made to the alertmanager spec [here](https://github.com/integr8ly/application-monitoring-operator/pull/156/files#diff-4be66ac58044fd8db91c27c657ba4c2be73e29eed2c70721ea95842f2e2769ebL9-R9).
